Source/JavaScriptCore/inspector/scripts/codegen/generate_objc_protocol_type_conversions_header.py

```diff
@@ -166,7 +166,7 @@ def _generate_enum_from_protocol_string(self, objc_enum_name, enum_values):
         lines.append('inline std::optional<%s> fromProtocolString(const String& value)' % objc_enum_name)
         lines.append('{')
         for enum_value in enum_values:
-            lines.append('    if (value == "%s")' % enum_value)
+            lines.append('    if (value == "%s"_s)' % enum_value)
```

Comment:
Would be better later to use `SortedArrayMap` for this.

Reply:
Will update.